pkg/util/kubernetes/kubelet/podwatcher.go

```diff
@@ -163,21 +160,12 @@ func (w *PodWatcher) Expire() ([]string, error) {
 		// pod was removed from the pod list, we can safely cleanup everything
 		if now.Sub(lastSeen) > w.expiryDuration {
 			delete(w.lastSeen, id)
-			delete(w.lastSeenReady, id)
 			delete(w.tagsDigest, id)
 			delete(w.oldPhase, id)
 			delete(w.oldReadiness, id)
 			expiredContainers = append(expiredContainers, id)
 		}
 	}
-	for id, lastSeenReady := range w.lastSeenReady {
-		// we keep pods gone unready for 25 seconds and then force removal
```

Comment:
do you have an explanation about this logic of waiting 30sec? 
I'm trying to get it from my memory, but I don't remember.